internal/templates/templates.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2023 Jetpack Technologies Inc and contributors. All rights reserved.
+// Use of this source code is governed by the license in the LICENSE file.
+
+package templates
+
+var templates = map[string]string{
+	"csharp":  "examples/development/csharp/hello-world",
+	"elixir":  "examples/development/elixir",
+	"fsharp":  "examples/development/fsharp/hello-world",
+	"go":      "examples/development/go/hello-world",
+	"haskell": "examples/development/haskell",
+	"java":    "examples/development/java",
+	"nim":     "examples/development/nim/spinnytest",
+	"nodejs":  "examples/development/nodejs",
+	"php":     "examples/development/php/php8.1",
+	"python":  "examples/development/python",
+	"ruby":    "examples/development/ruby",
+	"rust":    "examples/development/rust",
+	"zig":     "examples/development/zig",
```

Comment:
I'd recommend double-checking each of these. For example, I think these need to be:
```
	"rust":    "examples/development/rust/rust-stable-hello-world",
 	"zig":     "examples/development/zig/zig-hello-world",
```

arguably, maybe these need to be renamed to just `hello-world`, but then we should also update the devbox.sh landing page to have correct links

Reply:
I'll audit the list and see if there's anything I should add. @Lagoja would be great to have your input.